backend/blueprints/auth.py

```diff
@@ -0,0 +1,107 @@
+from datetime import datetime, timedelta
+
+from backend.extensions import db
+from backend.models import User, JWTToken
+from backend.serializers.login_serializer import LoginSchema
+from flask import Blueprint, current_app as app, jsonify, request
+from flask_jwt_extended import (
+    create_access_token, create_refresh_token, decode_token,
+    get_jwt_identity, jwt_required, jwt_optional, get_raw_jwt
+)
+from marshmallow import ValidationError
+
+
+auth_blueprint = Blueprint('auth', __name__, url_prefix='/auth')
+
+
+@auth_blueprint.route('/login', methods=['POST'])
+@jwt_optional
+def login():
+    current_user = get_jwt_identity()
+
+    if current_user:
+        return jsonify({"msg": "User already logged in as {}".format(
+            current_user)}), 401
+
+    if not request.is_json:
+        return jsonify({"msg": "No input data provided"}), 400
+
+    schema = LoginSchema()
+    try:
+        result = schema.load(request.json)
+    except ValidationError as error:
+        return jsonify({"msg": "Wrong input data",
+                        "errors": error.messages}), 400
+
+    username = result['username']
+    password = result['password']
+
+    if not (username and password):
+        return jsonify({"msg": "Username and password required"}), 400
+
+    user = User.query.filter_by(username=username).first()
+
+    if user and user.check_password(password):
+        access_token = create_access_token(
+            identity=username, expires_delta=timedelta(minutes=60))
+        refresh_token = create_refresh_token(
+            identity=username)
+
+        add_token_to_database(access_token, app.config['JWT_IDENTITY_CLAIM'])
+        add_token_to_database(refresh_token, app.config['JWT_IDENTITY_CLAIM'])
+
+        ret = {
+            'access_token': access_token,
+            'refresh_token': refresh_token
+        }
+        return jsonify(ret), 201
+    else:
+        return jsonify({"msg": "Not authorized"}), 401
+
```

Comment:
We maybe should create separate issue for refresh_token, and all related to it things like refreshing endpoint, and refresh logout?